From fc133aa8647fe34a41c70bd5915c7f7f655d3160 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Christian=20Gr=C3=BCnhage?=
 <jan.christian@gruenhage.xyz>
Date: Thu, 7 Oct 2021 18:05:15 +0200
Subject: [PATCH 2/3] move sbin/* into bin/*

Void does not distinguis between bin and sbin, but this patch will
likely not be upstreamable.
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index d4c241e..2694e78 100644
--- a/Makefile
+++ b/Makefile
@@ -48,7 +48,7 @@ scripts: scripts/beesd scripts/beesd@.service
 install_tools: ## Install support tools + libs
 install_tools: src
 	install -Dm755 bin/fiemap $(DESTDIR)$(PREFIX)/bin/fiemap
-	install -Dm755 bin/fiewalk $(DESTDIR)$(PREFIX)/sbin/fiewalk
+	install -Dm755 bin/fiewalk $(DESTDIR)$(PREFIX)/bin/fiewalk
 
 install_bees: ## Install bees + libs
 install_bees: src
@@ -56,7 +56,7 @@ install_bees: src
 
 install_scripts: ## Install scipts
 install_scripts: scripts
-	install -Dm755 scripts/beesd $(DESTDIR)$(PREFIX)/sbin/beesd
+	install -Dm755 scripts/beesd $(DESTDIR)$(PREFIX)/bin/beesd
 	install -Dm644 scripts/beesd.conf.sample $(DESTDIR)/$(ETC_PREFIX)/bees/beesd.conf.sample
 ifneq ($(SYSTEMD_SYSTEM_UNIT_DIR),)
 	install -Dm644 scripts/beesd@.service $(DESTDIR)$(SYSTEMD_SYSTEM_UNIT_DIR)/beesd@.service
-- 
2.35.1

